src/task 25/main.js

```diff
@@ -0,0 +1,10 @@
+var arr = ["81","87","88"];
+var field = document.querySelector("input");
+field.addEventListener("keydown", function(event) {
+    for(i = 0 ; i <= arr.length; i++){
+        if (event.keyCode == arr[i]){
+            event.preventDefault();
+        };
```

Comment:
You can use indexOf method. This will allow you not to iterate through the array on code level at all.